### PR TITLE
Add AWS Foundational Security Best Practices CloudFront benchmark. Closes #129

### LINKED ISSUE #129: Add AWS Foundational Security Best Practices CloudFront benchmark
**Is your feature request related to a problem? Please describe.**
I'd like to be able to run the CloudFront benchmark for the AWS Foundational Security Best Practices framework.

**Describe the solution you'd like**
Implement CloudFront benchmark.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [x] Issue(s) linked

### CODE REVIEW

File: query/cloudfront/cloudfront_distribution_encryption_in_transit.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  b.arn as resource,
+  case
+    when d.arn is not null or (default_cache_behavior ->> 'ViewerProtocolPolicy' = 'allow-all') then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when d.arn is not null or (default_cache_behavior ->> 'ViewerProtocolPolicy' = 'allow-all')  then title || ' data not encrypted in transit.'
+    else title || ' data encrypted in transit.'
+  end as reason,
+  -- Additinal Dimensions
```
Comment (cbruno10) [inline query/cloudfront/cloudfront_distribution_encryption_in_transit.sql:25]:
```suggestion
  -- Additional Dimensions
```

I see this in a few other places, can you please fix it there too?

File: query/cloudfront/cloudfront_distribution_encryption_in_transit.sql [hunk 2]
```diff
@@ -0,0 +1,31 @@
+with data as (
```
Comment (cbruno10) [inline query/cloudfront/cloudfront_distribution_encryption_in_transit.sql:1]: `query/cloudfront/cloudfront_distribution_encryption_in_transit_enabled.sql`

File: query/cloudfront/cloudfront_distribution_configured_with_origin_failover.sql [hunk 3]
```diff
@@ -0,0 +1,16 @@
+select
+  -- Required Columns
+  arn as resource,
+  case
+    when origin_groups ->> 'Items' is not null then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when origin_groups ->> 'Items' is not null then title || ' origin group is configured.'
```
Comment (cbruno10) [inline query/cloudfront/cloudfront_distribution_configured_with_origin_failover.sql:9]: The query name and ok/alarm messages are a bit different, does having an origin group configured mean failover is also enabled? Can you please link to the relevant docs on this feature?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments

File: query/cloudfront/cloudfront_distribution_configured_with_origin_failover.sql (hunk 3 above)
Comment (khushboo9024) [reply query/cloudfront/cloudfront_distribution_configured_with_origin_failover.sql:9]: https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html#fsbp-cloudfront-4
```
This control checks whether an Amazon CloudFront distribution is configured with an origin group that has two or more origins.
```

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**